README.md

```diff
@@ -1,3 +1,66 @@
+  - `<year>`: Short version of the year the entry was published in. It has only two digits, ommitting the first two digits of a year number (i.e., `96` for 1996 and `05` for 2005). If you get old enough to find a redundancy caused by this two-digit abbreviation: congratulations.
+  - In case that the generated key is conflicting with an existing key, add a suffix, such as `b`, `c`, ...
+- **If a journal does not have an official abbreviation** of letters only we still use our custom abbreviation for MYshort.bib. For example, the _Journal on Software Maintenance: Research and Practice_ should be abbreviated _J. Softw. Maint: Res. Pract._ but not by its variable name `JSMRP` we invented. Still, we abbreviate it in MYshort with `@String{JSMRP = "JSMRP"}` for consistency.
+- **If you detect an ill-formed key**, please update it. Leave a note that you did so including the previous key such that authors of papers using the old key can quickly find the updated key:
+    ```tex
+    % formerly known as KAK08
+    @inproceedings{KAK:ICSE08,
+    	author = {Christian K\"{a}stner and Sven Apel and Martin Kuhlemann},
+    	title = {{Granularity in Software Product Lines}},
+    	booktitle = ICSE,
+    ...
+    ```
+
+### Policies for Theses
+
+- **Keys** of theses are formatted according to the formatting rules described above.
+- **The month of a thesis entry** is the month of the defense (because publication of a thesis might happen much later).
+- **Publication status of theses** should be indicated in exactly one of the following three ways:
+  - If the thesis is already published, add DOI and URL. The URL should directly point to the PDF.
+  - If the thesis is not published and should not be published, add `% will not be published` as a comment above the entry.
+  - If the thesis is not published but should be published, add `note = {To appear}`.
+- **Add `type` for bachelor, master, and project thesis**. These theses should be specified as `@mastersthesis` as it is the only bibtex entry type for thesis (apart from `@phdthesis`). Distinguish between different types of thesis by using `type = {Bachelor's Thesis}`, `type = {Master's Thesis}`, or `type = {Project Thesis}` respectively.
+- **For theses written in german** add `note = {In German}`.
+
+## Custom Fields
+Use custom fileds to add addtional information for single entries. **Do not write comments inbetween entries.**
+- **comments**: Contains miscellaneous comments for this entry.
+  - Common strings:
+    - NotToBePublished (For Bachelor/Master theses)
+  - Example: `comments = NotToBePublished,`
+
+- **missing**: Documents which information for required fields are unavailable for this entry.
+  - Common strings:
+    - NoDOI
+    - NoUrl
+    - NoPages
+    - NoMonth
+    - NoPublisher
+    - NoAddress
+    - NoChapterNo
+  - Example: `missing = NoDOI # NoURL # NoMonth # NoAddress,`
```

Comment:
Is this correct syntax with the #? Shouldn#t it be `missing = {NoDOI, NoURL, NoMonth, NoAddress}`

Reply:
This is correct. # is concatenation within bibtex.
` {NoDOI, NoURL, NoMonth, NoAddress}` would treat these string as literals.